subscriber/podaac_access.py

```diff
@@ -287,6 +290,26 @@ def get_temporal_range(start, end, now):
     raise ValueError("One of start-date or end-date must be specified.")
 
 
+def download_file(remote_file, output_path, retries=3):
+    failed = False
+    for r in range(retries):
```

Comment:
For consistency should probably use `@tenacity.retry`.
Check out https://github.com/podaac/data-subscriber/pull/92/files/5ad8e149843d5499b8601740cbc963aaac0f46f5..78aac3ffb4d6586fbc796935e85e67df41f5b53b#diff-158519578c6b1bf3dc62de68856136d011247d7ed6b085dfb341dfe452d2a1cfR314